src/main/java/com/spotify/docker/client/DockerClient.java

```diff
@@ -2845,4 +2847,16 @@ public ListVolumesFilterParam(String name, String value) {
    * @throws InterruptedException If the thread is interrupted
    */
   List<Node> listNodes() throws DockerException, InterruptedException;
+
+  /**
+   * Inspects a swarm node. Only available in Docker API &gt;= 1.24.s
+   *
+   * @param nodeId The id of the swarm node to inspect
+   * @return info about the node
+   *
+   * @throws NonSwarmNodeException if the node is not part of a swarm (503)
+   * @throws DockerException      if a server error occurred (500)
+   * @throws InterruptedException if the thread is interrupted
+   */
```

Comment:
Can you add `@since API 1.24`? I know you have it in the text above, but it is nice to keep the format consistent.